components/homme/src/theta/prim_advance_mod.F90

```diff
@@ -104,7 +104,8 @@ end subroutine vertical_mesh_init2
 
 
   !_____________________________________________________________________
-  subroutine prim_advance_exp(elem, deriv, hvcoord, hybrid,dt, tl,  nets, nete, compute_diagnostics)
+  subroutine prim_advance_exp(elem, deriv, hvcoord, hybrid,dt, tl,  nets, nete, compute_diagnostics, &
```

Comment:
See previous comment for `components/homme/src/pese/prim_advance_mod.F90`, does this mean single_column is not supported for the `theta` version of HOMME?